src/plugins/charts/common/index.ts

```diff
@@ -30,8 +26,14 @@ export {
   LabelRotation,
   defaultCountLabel,
   MULTILAYER_TIME_AXIS_STYLE,
-  checkIsMinContinuity,
-  checkIsMaxContinuity,
 } from './static';
 
-export type { ColorSchemaParams, Labels, Style, PaletteContinuity } from './types';
+export type { ColorSchemaParams, Labels, Style } from './types';
+
+/** @deprecated **/
```

Comment:
It seems like no one is importing these from the charts plugin at the moment - should be safe to remove.

Reply:
@flash1293 Not certainly in that way

See` src/plugins/charts/public/services/palettes/helpers.ts` line 9